### DISCUSSION TITLE
Globally email templates / workflows

### DISCUSSION THREAD

**Discussion by bjarnef** [umbraco/Umbraco.Forms.Issues#1518] [Ideas]
When working with workflows on many forms I often thing I need to do the same over and over and unfortunately is isn't possible to copy workflows (except the entire form): https://github.com/umbraco/Umbraco.Forms.Issues/issues/1185

I think it could be useful to define some globally email templates or workflow to re-use. Although workflows specific per form is great, we have mainly used the same configuration, which is tedious to manage when the number of forms grow, e.g. in a multi-site solutions where the customer may prefer a form per language/culture/site.

A concreate scenario it that I have a workflow "Send email to user" to send confirmation email to user (copy of form submission).

![image](https://github.com/umbraco/Umbraco.Forms.Issues/assets/2919859/df81b763-4601-418d-aa94-f2c4be046762)

An idea could be be to define this globally, but of course each form using this would need to have the field with alias `email`.

Another idea is just the email part, e.g. something similar to email templates in Umbraco Commerce:

- Email template (file)
- Header text
- Footer text
- Subject
- CC, BCC etc.

That way on the workflow configuration I only need to select the same email template (node). If the customer want to adjust header or footer text, it doesn't need to be done on 100 forms, just on a single node.

Perhaps consider is as global content similar to global blocks in Umbraco https://github.com/umbraco/rfcs/discussions/35

**Comment by TvGessel** [comment]
Duplicate with #1572
